### PR TITLE
First `versioned` version of curl is 7.74, not 7.48. Reset base version to 7.74.

### PR DESCRIPTION
**Pull request by LarryOsterman**
- [ ] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] SHA512s are updated for each updated download
- [ ] The "supports" clause reflects platforms that may be fixed by this new version
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [ ] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [ ] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by LarryOsterman** [comment]
Tagging @BillyONeal

**Comment by FrankXie05** [comment]
@LarryOsterman Is there any official statement about the minimum version support for `curl`?

**Comment by LarryOsterman** [comment]
> @LarryOsterman Is there any official statement about the minimum version support for `curl`?

Yes. The minimum version for the Azure SDK for C++ is 7.44.

However vcpkg will not let us express that, so I've been iterating trying to find ANY version of curl that we can mark as our minimum supported version. And thus this series of pull requests.

File: ports/azure-core-cpp/vcpkg.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
           "features": [
             "ssl"
           ],
-          "version>=": "7.48.0"
+          "version>=": "7.74.0"
```
Comment (dg0yt) [inline ports/azure-core-cpp/vcpkg.json:42]: Is `7.74.0` good enough, or does it have to be `7.74.0#5`?  (I don't know.)

Comment (LarryOsterman) [reply ports/azure-core-cpp/vcpkg.json:42]: I honestly don't know either. 

I'm just trying to find any version that works, to be honest.

**Comment by BillyONeal** [comment]
Given that any version of curl vcpkg is ever going to give complies with what Azure SDKs wants, should we just remove the version constraint entirely?

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: versions/a-/azure-core-cpp.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "versions": [
+    {
+      "git-tree": "db976e05847ef9b5f9a0ab9668f34add5cef3d79",
+      "version-semver": "1.10.2",
+      "port-version": 4
+    },
```
[the remark this answers is not in the document]
Comment (LarryOsterman) [reply versions/a-/azure-core-cpp.json:7]: This suggested change confuses me. If I remove the `},` won't that render the JSON invalid?

[the remark this answers is not in the document]
Comment (FrankXie05) [reply versions/a-/azure-core-cpp.json:7]: For the modification of a port, we require `port-version` +1.

[the remark this answers is not in the document]
Comment (LarryOsterman) [reply versions/a-/azure-core-cpp.json:7]: As best as I can figure, the suggested change removes line 7 from the file. Won't that break things?

[the remark this answers is not in the document]
Comment (dg0yt) [reply versions/a-/azure-core-cpp.json:7]: The suggestion is to remove the whole `port-version` 4 entry: A single PR can only a singe entry for each port.

**Review by FrankXie05 — APPROVED** [review]

**Pull request merged**